ouroboros-network/test/Test/Ouroboros/Network/Testnet/Simulation/Node.hs

```diff
@@ -0,0 +1,346 @@
+                AcceptedConnectionsLimit maxBound maxBound 0
+              diffusionMode = InitiatorAndResponderDiffusionMode
+              readPRP = return []
+              readULA = return (UseLedgerAfter 0)
+
+              acceptVersion = \_ v -> Accept v
+
+              defaultMiniProtocolsLimit :: MiniProtocolLimits
+              defaultMiniProtocolsLimit =
+                MiniProtocolLimits { maximumIngressQueue = 64000 }
+
+              blockGeneratorArgs :: BlockGeneratorArgs Block StdGen
+              blockGeneratorArgs =
+                randomBlockGenerationArgs bgaSlotDuration
+                                          bgaSeed
+                                          quota
+
+              stdChainSyncTimeout :: ChainSyncTimeout
+              stdChainSyncTimeout = do
+                  ChainSyncTimeout
+                    { canAwaitTimeout  = shortWait
+                    , intersectTimeout = shortWait
+                    , mustReplyTimeout
+                    }
+
+              limitsAndTimeouts :: Node.LimitsAndTimeouts Block
+              limitsAndTimeouts
+                = Node.LimitsAndTimeouts
+                    defaultMiniProtocolsLimit
+                    (byteLimitsChainSync (const 0))
+                    (timeLimitsChainSync stdChainSyncTimeout)
+                    defaultMiniProtocolsLimit
+                    (byteLimitsKeepAlive (const 0))
+                    timeLimitsKeepAlive
+                    defaultMiniProtocolsLimit
+                    (ProtocolSizeLimits (const smallByteLimit) (const 0))
+                    (ProtocolTimeLimits (const (Just 60)))
+                    defaultMiniProtocolsLimit
+                    (ProtocolSizeLimits (const (4 * 1440))
+                                        (fromIntegral . BL.length))
+                    (ProtocolTimeLimits (const shortWait))
+
+              interfaces :: Node.Interfaces m
+              interfaces =
+                Node.Interfaces ntnSnocket
+                                acceptVersion
+                                ((return .) <$> domainResolver raps dMap)
+                                ntcSnocket
+                                stdGen
+                                dMap
+                                (LedgerPeersConsensusInterface
+                                   $ \_ -> return Nothing)
+
+              arguments :: Node.Arguments m
+              arguments =
+                Node.Arguments rap
+                               acceptedConnectionsLimit
+                               diffusionMode
+                               0
+                               0
+                               peerSelectionTargets
+                               (return lrp)
+                               readPRP
+                               readULA
+                               5
+                               30
+                               dnsTimeout
+                               dnsLookupDelay
```

Comment:
Let's use named fields, easier to figure out what is what.

Thinks like, `aKeepAliveInterval`, `aPingPongInterval`, `aProtocolTimeout`, `aTimeWaitTimeout` should be defined at the top level. 

I would rename `dnsTimeout -> dnsTimeoutScript` and `dnsLookupDelay` to `dnsLookupDelayScript`.  It's not that much more to type, and it caries the meaning what it actually is.